### PR TITLE
Decompile bomber.c & mirra.c

### PR DESCRIPTION
**Pull request by Cleverking2003**

### CODE REVIEW

File: include/bomber.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+struct Object2* CreateBomber(struct Object*, u8);
+void sub_080C478C(struct Object2*);
+void sub_080C47EC(struct Object2*);
+void sub_080C485C(struct Object2*);
+void sub_080C48B8(struct Object2*);
+void sub_080C48DC(struct Object2*);
+void sub_080C4900(struct Object2*);
+void sub_080C4928(struct Object2*);
+void sub_080C4968(struct Object2*);
+
```
Comment (jiangzhengwenjz) [inline include/bomber.h:17]: put static function decl in `xxx.c`

File: include/mirra.h [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+#include "global.h"
+#include "object.h"
+
+struct Object2* CreateMirra(struct Object*, u8);
+void sub_080B10A8(struct Object2*);
+void sub_080B1174(struct Object2*);
+void sub_080B11C0(struct Object2*);
+void sub_080B1218(struct Object2*);
+void sub_080B1254(struct Object2*);
+
```
Comment (jiangzhengwenjz) [inline include/mirra.h:14]: put static function decl in `xxx.c`

File: src/mirra.c [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                            sub_080B1218(arg0);
+                            break;
+                        }
+                    }
+                }
+            }
+        }
+    }
+    else {
+       goto else_l;
```
Comment (jiangzhengwenjz) [inline src/mirra.c:62]: Not tail merge?

File: src/mirra.c [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+void sub_080B11C0(struct Object2* arg0) {
+    u8 unk56;
+    struct Unk_0888562C *res;
+    if (arg0->object->unk14 != 0) {
+        gUnk_020229D8 |= 1 << (arg0->object->unk14 - 1);
+    }
+    unk56 = arg0->base.unk56;
+    res = sub_080025AC(unk56, arg0->object->x >> 4, arg0->object->y >> 4);
+    sub_08001408(unk56, res, 0, 0);
```
Comment (jiangzhengwenjz) [inline src/mirra.c:92]: what if you merge them into a single statement?

**Review by jiangzhengwenjz — CHANGES_REQUESTED** [review]

File: src/mirra.c (hunk 3 above)
Comment (Cleverking2003) [reply src/mirra.c:62]: I couldn't get it to do tail merge

**Review by jiangzhengwenjz — APPROVED** [review]

**Pull request merged**